docker-compose.yml

```diff
@@ -1,25 +1,63 @@
-version: '2'
+version: '3.1'
 
 services:
+
     postgres:
-        image: postgres:9.5-alpine
+        image: postgres:11.6-alpine
+        container_name: postgres
+        hostname: postgres
+        restart: always
         environment:
-            POSTGRES_USER: zotonic
+           - POSTGRES_USER=zotonic
+           - POSTGRES_DB=zotonic
+           - POSTGRES_PASSWORD=zotonic
+           - PGDATA=/var/lib/postgresql/data/pgdata
         volumes:
-            - ./data:/data
-        working_dir: /data
+           - "./docker-data/postgresql:/var/lib/postgresql/data/pgdata"
```

Comment:
It's good practice to use a named volume instead of a local directory.

Reply:
This is where my Docker knowledge ends :-)  I copied the above from the docs of the postgres container.